jupyter_collaboration/websocketserver.py

```diff
@@ -58,21 +58,11 @@ async def clean(self):
         #         self.log.warning(msg)
         #         self.log.debug("Pending tasks: %r", pending)
 
+        self.stop()
         tasks = []
-        for name, room in list(self.rooms.items()):
-            try:
-                self.delete_room(name=name)
-            except Exception as e:  # Capture exception as room may be auto clean
-                msg = f"Failed to delete room {name}"
-                self.log.debug(msg, exc_info=e)
-            else:
-                tasks.append(room._broadcast_task)  # FIXME should be upstreamed
         if self.monitor_task is not None:
             self.monitor_task.cancel()
```

Comment:
Should we also use AnyIO for tasks managed by jupyter_collaboration?

Reply:
I suggest so, yes. These changes are only for supporting the latest ypy-websocket.